shared/.flowconfig

```diff
@@ -50,4 +49,4 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue
 
 [version]
-0.24.1
+0.25
```

Comment:
we should set this to 0.25.0 just so we don't get another case where npm updates to 0.25.1 and we get a flow error